src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,74 @@
-export const Pagination = () => {};
+import React from 'react';
+
+interface PaginationProps {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+}
+
+export const Pagination: React.FC<PaginationProps> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const totalPages = Math.ceil(total / perPage);
```

Comment:
Performance issue: The calculation of `totalPages` will be done on each render of the component. It would be more efficient to use a `useMemo` hook to calculate this value only when `total` or `perPage` change.